docs/_ug/commandSummary/tagCommands/untag.md

```diff
@@ -1,2 +1,2 @@
 <!-- markdownlint-disable-file first-line-h1 -->
-<b>untag n/TAG_NAME id/INDEX_LIST</b><br><br><u>Valid Examples:</u><br>untag n/Condiments id/1<br>untag n/Condiments id/1,2,4,8<br><br><u>Invalid Example:</u><br>untag Condiments 1,2
+`untag n/TAG_NAME id/INDEX`
```

Comment:
Ditto. Should we change to the following?

```suggestion
`untag n/TAG_NAME INDEX`
```

Reply:
Ditto. Will change after implemnetation is changed